lightning_client.go

```diff
@@ -1914,10 +1914,18 @@ func (p *ChannelClosedUpdate) CloseTxid() chainhash.Hash {
 // sending an EOF), we close the updates and error channel to signal that there
 // are no more updates to be sent.
 func (s *lightningClient) CloseChannel(ctx context.Context,
-	channel *wire.OutPoint, force bool,
-	confTarget int32) (chan CloseChannelUpdate, chan error, error) {
+	channel *wire.OutPoint, force bool, confTarget int32,
+	addr btcutil.Address) (chan CloseChannelUpdate, chan error,
```

Comment:
nit: I'd call `addr` => `deliveryAddr` instead to make it more clear. (And/or extend godoc).

Reply:
renamed + added godoc :)